appcli/commands/configure_cli.py

```diff
@@ -98,25 +105,46 @@ def set(ctx, setting, value):
             configuration.save()
 
         @configure.command(help="Applies the settings from the configuration.")
+        @click.option(
+            "--message",
+            "-m",
+            help="Message describing the changes being applied.",
+            default="Commit via appcli.",
+            type=click.STRING,
+        )
```

Comment:
Turns out this was really simple and didn't need the code change we discussed today:
```
if cli_context.subcommand_args is not None:
            command.extend(f'"{x}"' for x in cli_context.subcommand_args)
```
The reason this wasn't originally working for me was because I have a launcher script which was:
```
./src/teraflow --debug -c /tmp/tera/c -d /tmp/tera/d $@
```
I believe the $@ was splitting my quoted strings up into separate arguments.
I am able to call the following to set my message successfully:
`./src/teraflow --debug -c /tmp/tera/c -d /tmp/tera/d configure apply -m "my message"`

Reply:
That's odd, the $@ should respect quoted strings. We'll need to keep an eye on this.

Reply:
launcher script that I was using:
```
#!/bin/bash
./src/teraflow --debug -c /tmp/tera/c -d /tmp/tera/d $@
```